lib/mastodon/cli/media.rb

```diff
@@ -151,7 +151,40 @@ def remove_orphans
           end
         end
       when :fog
-        fail_with_message 'The fog storage driver is not supported for this operation at this time'
+        fog_directory = Paperclip::Attachment.default_options[:fog_directory]
+        connection = Fog::Storage.new(Paperclip::Attachment.default_options[:fog_credentials])
+        directory = connection.directories.get(fog_directory)
+        last_key = options[:start_after]
+
+        loop do
+          objects = begin
+            directory.files.all(prefix: prefix, marker: last_key, limit: 1000).to_a
+          rescue => e
+            progress.log(pastel.red("Error fetching list of files: #{e}"))
+            progress.log("If you want to continue from this point, add --start-after=#{last_key} to your command") if last_key
+            break
+          end
+
+          break if objects.empty?
+
+          last_key = objects.last.key
+
+          objects.each do |object|
+            path_segments = object.key.split('/')
+            progress.increment
+            next unless orphaned_file?(path_segments)
```

Comment:
What is confusing is that the linter annotation is displayed at the wrong position.

The issue is here, and would be fixed by the following newline:
```suggestion
            next unless orphaned_file?(path_segments)

```

Reply:
Got it. Fixed!